VERSIONING.md

```diff
@@ -33,3 +33,35 @@ Compose version | Kotlin version
 1.0.0 | 1.5.31
 1.1.1 | 1.6.20
 1.2.0 | 1.7.10 - 1.7.20 (Desktop, Android), 1.7.10 (JS)
+
+### Relation to Jetpack Compose and Compose Multiplatform release cycle
+
+Compose Multiplatform shares a lot of code with [Jetpack Compose](https://developer.android.com/jetpack/compose) for Android developed by Google.
+We keep our release cycles aligned, so the common part is properly tested and stabilized.
+
+When the new minor version of Jetpack Compose is released, we pick the release commit, use it as a base for [Compose Multiplatform fork](https://github.com/JetBrains/androidx), finish new platform features, stabilize all platforms, and release Compose Multiplatform after some time.
```

Comment:
> It's probably "a new minor"

Fixed

> release after each minor release of Jetpack Compose

https://semver.org describes Major as the first number of the version, and Minor as the second. But to avoid confusion, I removed "major".

> Overall could be rephrased to something like:

My points are mostly subjective, but:
- "Compose Multiplatform shares a lot of code with Jetpack Compose" is better than "Compose Multiplatform is built on top of Jetpack Compose"
- I would like to not mention upstreaming process. Upstreaming has two parts - upstreaming common features and upstreaming desktop features. First may be relevant for users, but we need to think more how to properly describe it, without adding complexity, and the second probably isn't much relevant, it just our "implementation details"